### DISCUSSION TITLE
Discussion regarding point system for resumes

### DISCUSSION THREAD

**Discussion by ashutosh-rath02** [ashutosh-rath02/git-re#133] [Ideas]
Hey contributors, git-re has gone to great length in GSSoC. We are currently planning to add a new rating system for the resume. It can be a star system(out of 5 stars), point system /10, or we can think of our own rating system. I welcome you all to provide your ideas💡.

**Comment by Chelseasingla1** [comment]
> Hey contributors, git-re has gone to great length in GSSoC. We are currently planning to add a new rating system for the resume. It can be a star system(out of 5 stars), point system /10, or we can think of our own rating system. I welcome you all to provide your ideas💡.

How about levels like Beginner, Intermediate, Proficient, and Expert and this will i guess guide users too what type of job they can apply based on their profiles

**Reply by ashutosh-rath02** [reply]
Yeah we can keep that according to range of points, good one. But how are we going to decide the points? Any suggestions?

**Reply by Chelseasingla1** [reply]
Like by experience of the user, any hackathons or competition wins, number of work experience years,

**Reply by ratishjain12** [reply]
add badges for each range or something

**Reply by Chelseasingla1** [reply]
yeah basically we can give stars to each skill too, like where they stand in a particular skill, similarly giving a range to experience level.

**Comment by NilanchalaPanda** [comment]
Currently we are thinking to unify the rating system, what if we divide the rating system on basis on certain parameters. On then combine them to give a `GitScore` (Just a catchy name), that represents the overall rating of the resume.

The divided rating can consists of on basis of :
- Years of Exp (Can be student, working, expert. Basically categorical classification).
- Technical Proficiency.
- Projects.
- Contributions.

Please add any more points you think would be relevant.
